From cf7c4a5c89c35a6310bc535a41ee12b163691ccb Mon Sep 17 00:00:00 2001
From: Weiwei Wang <weiwei.wang@windriver.com>
Date: Tue, 1 Nov 2011 10:36:44 +0800
Subject: [PATCH] core: Rename Cert Profile to Safety Profile at guest side

Cert Profile is being renamed to Safety Profile in Hyp 2.0,
guest linux need to do the adjustments accordingly.

Signed-off-by: Weiwei Wang <weiwei.wang@windriver.com>
---
 init/Kconfig.wrhv |    6 +++---
 1 files changed, 3 insertions(+), 3 deletions(-)

diff --git a/init/Kconfig.wrhv b/init/Kconfig.wrhv
index 72f60bd..ec86c28 100644
--- a/init/Kconfig.wrhv
+++ b/init/Kconfig.wrhv
@@ -8,12 +8,12 @@ config WRHV
 	select SERIAL_CORE
 	select SERIAL_CORE_CONSOLE
 
-config WRHV_CERT
-	bool "Certifiable Hypervisor"
+config WRHV_SAFETY_PROFILE
+	bool "Safety Profile Hypervisor"
 	default n
 	depends on WRHV
 	help
-	  This options select certifiable hypervisor virtualization extensions
+	  This options select safety profile hypervisor virtualization extensions
 	  in the Linux Kernel
 	select WRHV_COREVBI_ONLY
 
-- 
1.7.0.4

